data/mona/meta.yaml

```diff
@@ -0,0 +1,136 @@
+      description: The entropy of the spectrum.
+    - id: normalized_entropy
+      type: continuous
+      units:
+      names:
+          - noun: normalized entropy
+      description: The normalized entropy of the spectrum (ratio of spectral entropy to maximum possible entropy for a spectrum with the same number of
+          peaks).
+identifiers:
+    - id: smiles
+      type: SMILES
+      description: SMILES
+    - id: inchi
+      type: InChI
+      description: InChI
+    - id: inchikey
+      type: InChIKey
+      description: InChIKey
+    - id: id
+      type: Other
+      description: MassBank ID
+      sample: 'False'
+templates:
+    - prompt: |-
+          Please answer the following chemistry question.
+          What is the spectral entropy of the following mass spectrum?
+          <spectrum#text>
+      completion: <spectral_entropy#value>
+    - prompt: |-
+          Please answer the following chemistry question.
+          Which of the molecules, <molecule1#text> or <molecule2#text>, is more likely to produce the following mass spectrum?
+          <spectrum#text>
+      completion: <molecule1#text>
+fields:
+    exp_values:
+        values:
+            - name: spectrum
+              column: spectrum
+              text: Raw mass spectrum represented as a set of (m/z location, intensity) pairs
+            - name: spectral_entropy
+              column: spectral_entropy
+              text: The entropy of the spectrum
+            - name: normalized_entropy
+              column: normalized_entropy
+              text: The normalized entropy of the spectrum (ratio of spectral entropy to maximum possible entropy for a spectrum with the same number of
+                  peaks)
+    metadata:
+        values:
+            - name: id
+              column: id
+              text: MassBank ID
+            - name: score
+              column: score
+              text: Quality score of the spectrum (1-5, 1 being low and 5 being high)
+            - name: library
+              column: library
+              text: Library the spectrum was obtained from
+            - name: molecular_formula
+              column: molecular_formula
+              text: Molecular formula
+            - name: accession
+              column: accession
+              text: Accession number
+            - name: date
+              column: date
+              text: Date of upload
+            - name: license
+              column: license
+              text: License
+            - name: instrument
+              column: instrument
+              text: Specific model of mass spectrometer
+            - name: instrument_type
+              column: instrument_type
+              text: General type of mass spectrometer
+            - name: ms_level
+              column: ms_level
+              text: MS level for MSn data (MS1-MS5, can also be composite)
+            - name: ionization_mode
+              column: ionization_mode
```

Comment:
do you think we should include this in the prompts?

Reply:
Hmmm. Not sure where the `mona` folder came from. Is this merging with my PR?